geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderQueue.java

```diff
@@ -1590,7 +1621,13 @@ public void cleanUp() {
 
   @Override
   public void close() {
-    // Because of bug 49060 do not close the regions of a parallel queue
+    Region r = getRegion();
+    if (!this.asyncEvent && r != null && !r.isDestroyed()) {
+      try {
+        r.close();
+      } catch (RegionDestroyedException e) {
```

Comment:
We catch a region destroyed exception here, do we need the !r.isDestroyed() check in the if block then?

Reply:
Removed